### PR TITLE
Add status field to assets table

### PR DESCRIPTION
**Pull request by istarkov**
## Description

I need this filed in the following PR, to count assets before upload. And restrict the total number of assets per project.

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)

## Before requesting a review

- [x] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-designer/blob/main/apps/designer/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `designer/env-check.js` if mandatory

### CODE REVIEW

File: packages/prisma-client/prisma/schema.prisma [hunk 1]
```diff
@@ -22,6 +22,11 @@ enum Location {
   REMOTE
 }
 
+enum UploadStatus {
```
Comment (istarkov) [inline packages/prisma-client/prisma/schema.prisma:25]: Do we name enums as CAPITALIZED?
Just one enum exists hard to decide

Comment (kof) [reply packages/prisma-client/prisma/schema.prisma:25]: prisma uses capitalized, so I would do the same, since I have no opinion on this

**Review by kof — APPROVED** [review]

File: packages/prisma-client/prisma/migrations/20221227220622_assets-status/migration.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+-- CreateEnum
+CREATE TYPE "UploadStatus" AS ENUM ('uploading', 'uploaded');
+
+-- AlterTable
+ALTER TABLE "Asset" ADD COLUMN     "status" "UploadStatus" NOT NULL DEFAULT 'uploaded';
```
Comment (TrySound) [inline packages/prisma-client/prisma/migrations/20221227220622_assets-status/migration.sql:5]: Why so many spaces?

Comment (istarkov) [reply packages/prisma-client/prisma/migrations/20221227220622_assets-status/migration.sql]: Its generated by prisma, have no idea

**Pull request merged**